### PR TITLE
More edits

### PR DESCRIPTION
**Pull request by ycaokris**
Closes Issue https://github.com/plotly/dash-oil-gas-ternary/issues/2

Changes include:

- Set default dragmode, disable legend toggling  to lasso for upper two plots
- Allow production visualization by legend-click on bar plot
- Minor styling on production plot

Deployment: 
https://dash-playground.plotly.host/GIT/dash-oil-gas-ternary 

https://dash-gallery.plotly.host/dash-oil-gas-ternary/

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def generate_production_plot(processed_data):
             title='Year'
         ),
         yaxis=dict(
-            title='GAS Production(mcf)'
+            title='GAS Production(mcf)',
```
Comment (shammamah) [inline app.py]:
```suggestion
            title='GAS Production (mcf)',
```

File: app.py [hunk 2]
```diff
@@ -377,7 +387,7 @@ def get_selection_by_bar(bar_selected_data):
                         build_graph_title('Select Operator'),
                         dcc.Dropdown(
                             id='operator-select',
-                            options=[{'label': i, 'value': i} for i in df['op'].unique().tolist()],
+                            options=[{'label': i, 'value': i} for i in df['op'].unique().tolist()[:-1]],
```
Comment (shammamah) [inline app.py]: Why aren't you taking the last item in the list here?

File: app.py [hunk 3]
```diff
@@ -388,9 +398,9 @@ def get_selection_by_bar(bar_selected_data):
                     className='five columns',
                     children=html.P(
                         className='instructions',
-                        children="Select data points from well map, ternary map or bar graph to visualize cross-filter "
-                                 "in other plots. Selection could be done by clicking on well map data point/individual bar, or use "
-                                 "lasso/box tool from modebar, multiple regions could be selected by holding SHIFT "
+                        children="Select data points from well map, ternary map or bar graph to visualize cross-filter to"
```
Comment (shammamah) [inline app.py]:
```suggestion
                        children="Select data points from the well map, ternary map, or bar graph to visualize cross-filtering to"
```

File: app.py [hunk 4]
```diff
@@ -388,9 +398,9 @@ def get_selection_by_bar(bar_selected_data):
...
+                                 " other plots. Selection could be done by clicking/lasso-circle on individual data points/bar, or use "
```
Comment (shammamah) [inline app.py]:
```suggestion
                                 " other plots. Selection could be done by clicking on individual data points or using the lasso tool to capture multiple data points or bars. With the"
```

File: app.py [hunk 5]
```diff
@@ -388,9 +398,9 @@ def get_selection_by_bar(bar_selected_data):
...
+                                 "box tool from modebar, multiple regions could be selected by holding SHIFT "
```
Comment (shammamah) [inline app.py]:
```suggestion
                                 "box tool from modebar, multiple regions can be selected by holding SHIFT "
```

File: assets/oil-and-gas-ternary.css [hunk 6]
```diff
@@ -49,4 +49,18 @@
...
     font-style: italic;
     font-family: 'Open Sans', sans-serif;
-}
\ No newline at end of file
+}
+
+
+/* this will prevent traces from toggling from the legend
+
+/* hides the svg dom element that has the click handler responsible for toggling */
```
Comment (shammamah) [inline assets/oil-and-gas-ternary.css]: Nice!

File: app.py (hunk 2 above)
Comment (ycaokris) [reply app.py]: ah thanks for point it out. This actually comes from a line separator added to the end of csv in previous commits (https://github.com/plotly/dash-oil-gas-ternary/commit/551a4d366c7519d0e4a0be465e633a8b61d725f9), which leads to a 'nan' for last item. It's generated by a `git config --global core.autocrlf` setting in my home device. Shouldn't be generated by scripts and will remove.

File: app.py (hunk 1 above)
Comment (ycaokris) [reply app.py]: 💯

File: app.py (hunk 3 above)
Comment (ycaokris) [reply app.py]: 💯

File: app.py (hunk 5 above)
Comment (ycaokris) [reply app.py]: 💯

**Comment by ycaokris** [comment]
@shammamah  Thanks for reviewing 👍

**Pull request merged**